src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -21,8 +21,13 @@ export default function withAnnotations<P extends object>(
     class ComponentWithAnnotations extends React.Component<P & WithAnnotationsProps, AnnotatorState> {
         static displayName: string;
 
+        handleActiveChange = (annotationId: string | null): void => {
```

Comment:
Can this live next to the other handler, as before?

Reply:
I moved it here because TS was complaining that I attached it to the handler before it was declared. But some other feedback from Conrad for the event design will change this slightly.